CONTRIBUTING.md

```diff
@@ -133,3 +134,17 @@ During the PR process, expect that there will be some back-and-forth. Please try
 If we accept the PR, a [maintainer](MAINTAINERS.md) will merge your change and usually take care of backporting it to appropriate branches ourselves.
 
 If we reject the PR, we will close the pull request with a comment explaining why. This decision isn't always final: if you feel we have misunderstood your intended change or otherwise think that we should reconsider then please continue the conversation with a comment on the PR and we'll do our best to address any further points you raise.
+
+## Feature branches
+
+For long running features that need collaboration between multiple members of the community, we use feature branches. We use a hybrid of the feature branch development model and the continuous integration development model. Maintainers of a repository can create a feature branch and a corrsponding label associated with the feature. Each feature branch will have the following safeguards:
```

Comment:
Would you please point the direction(might be links) to guide the contributors towards finding what is the best way to ask the maintainers to create the feature branch? Should they create an issue and request maintainers? Please provide more details.

Reply:
Good callout, i will add a sentence for contributors looking to introduce a new feature